packages/cli/src/commands/integration/open-integration.ts

```diff
@@ -0,0 +1,58 @@
+import chalk from 'chalk';
+import open from 'open';
+import type Client from '../../util/client';
+import getScope from '../../util/get-scope';
+import type { Configuration } from './types';
+import { fetchMarketplaceIntegrations } from './client';
+
+export async function openIntegration(client: Client, args: string[]) {
+  if (args.length > 1) {
+    client.output.error('Cannot open more than one dashboard at a time');
+    return 1;
+  }
+
+  const integrationSlug = args[0];
+
+  if (!integrationSlug) {
+    client.output.error('You must pass an integration slug');
+    return 1;
+  }
+
+  const { team } = await getScope(client);
+
+  if (!team) {
+    client.output.error('Team not found');
+    return 1;
+  }
+
+  let configuration: Configuration | undefined;
+
+  try {
+    configuration = (
+      await fetchMarketplaceIntegrations(client, integrationSlug)
+    )[0];
+  } catch (error) {
+    client.output.error(
+      `Failed to fetch configuration for ${chalk.bold(`"${integrationSlug}"`)}: ${(error as Error).message}`
+    );
+    return 1;
+  }
+
+  if (!configuration) {
+    client.output.error(
+      `No configuration found for ${chalk.bold(`"${integrationSlug}"`)}.`
+    );
+    return 1;
+  }
+
+  client.output.print(
+    `Opening the ${chalk.bold(integrationSlug)} dashboard...`
+  );
+
+  const url = new URL('/api/marketplace/sso', 'https://vercel.com');
```

Comment:
So, the goal is to SSO to the partner right away? I wonder if it'd be better to open our page and SSO from there? Just curious to hear thoughts: either way is possible.

Reply:
Correct. In the original conversation thread, the interest seemed to be in opening Vercel page for the given integration, but then in the more up-to-date technical outline the desired functionality was to deep-link all the way to the provider's page.

I'm personally more in-favour of deep linking. While opening to the Vercel page would be useful for cases like upgrading a plan sometimes, or viewing usage data through our tabs, more often than not a user doing their day-to-day development will want to deal with the resource directly, which they'd do through the provider's portal rather than ours. A developer using the CLI would more likely just go straight from the Vercel dashboard to the provider's dashboard, so we're saving a click in the common use case.

Reply:
Ok. I'm ok with that in general, but just to give it one more discussion in case we can leave some room open. One caveat that's specific to our dashboard: a resource exists in the context of an account (a team), but it's also linked to the project. And that linking to projects is unique to us and could be very relevant in the context of CLI, because it shows env vars for the project. WDYT?

Reply:
This is an interesting and specific one, but I feel like it feeds back into the other PR comment you made on how the user knows which team they're in the context of when using the CLI.

I do expect, however, that #12251 will help to enable the user's understanding of that project-to-resource relationship.

One line we could consider, would be a future addition of an option `--vercel` to open the integration's Vercel dashboard page instead of the integration provider's page. This would enable the flow to the data you're talking about, without disrupting the typical expected flow.